### ISSUE TITLE
Add helper text for Request Update on rollout

### ISSUE THREAD

**Issue by eliserichards** [mozilla/experimenter#10141]
On a live Rollout, when the population % hasn't been changed, we grey out the "Request Update" button since there is no update to request:
<img width="835" alt="image" src="https://github.com/mozilla/experimenter/assets/43795363/96159fb5-7c50-4bc5-8212-72aaaf735f45">

Let's add a little helper text when they hover over the button to make it clear _why_ it is greyed out and what they can do to change it.

"In order to update, update your population percent on the Audience page" (or something like that).

**Comment by enochval** [comment]
@eliserichards can I work on this?

**Comment by eliserichards** [comment]
@enochval Yes please! I will assign it to you 😄 Take a look at our [contributing doc](https://github.com/mozilla/experimenter/blob/main/contributing.md) for some helpful info, and if you get stuck you can reach out here or in the  [Nimbus channel](https://matrix.to/#/#nimbus:mozilla.org) on [Matrix](https://wiki.mozilla.org/Matrix)

**Comment by olamidepeterojo** [comment]
@eliserichards i would love to work on this but I'm stuck and would like to know how to test the changes i make to the button

**Comment by hariprabhu571** [comment]
can you assign this to me?
